### PR TITLE
fix: Prayer Experience Landing Page Avatars

### PR DESCRIPTION
**Pull request by calebpanza**
## ✏️ Description
When the user lands on the Daily Prayer Experience, they should see a preview of the Avatars **of the prayer requests they are about to see**. This is really important to note that these are NOT the same Avatars that show up on the Daily tab. 

**Daily Tab Prayer Experience Card:** people who have recently completed the Prayer Experience
**Prayer Welcome:** people who have recently _submitted prayer requests_

## 🔬 To Test
1. Load up the simulator
2. Open up the Daily tab
3. Press in to the Daily Prayer Experience

## 📸 Screenshots
https://user-images.githubusercontent.com/45076058/205751438-c48678b1-7d72-4ca6-ac63-50f0772501d6.mp4

## ⚠️ To-do Before Merge
n/a

## 📝 Checklist

- [x] I have performed a self-review of my code
- [x] I have tested my code works in the project (nothing breaks)
- [x] I have commented my code in hard-to-understand areas
- [x] New and existing tests and linting pass locally with my changes
- [x] I have made corresponding changes to the documentation (new envs?)
- [x] For PR's with design updates/changes, I have consulted with the product designer and made all necessary adjustments in Figma

### CODE REVIEW

File: templates/mobile/src/ui-kit/components/AvatarBlob/AvatarBlob.styles.js [hunk 1]
```diff
@@ -46,8 +46,8 @@ export const Leaf = styled.View(({ leaf, parentWidth, parentHeight }) => {
 });
 
 export const Avatar = styled(BaseAvatar)(({ leaf }) => ({
-  height: leaf.r * 2,
-  width: leaf.r * 2,
+  height: (leaf?.r || 0) * 2,
```
Comment (dsilva609) [inline templates/mobile/src/ui-kit/components/AvatarBlob/AvatarBlob.styles.js:49]: is this still valid if height or width end up being 0?

Comment (dsilva609) [reply templates/mobile/src/ui-kit/components/AvatarBlob/AvatarBlob.styles.js:49]: image
I get this if both are 0

Comment (camrun91) [reply templates/mobile/src/ui-kit/components/AvatarBlob/AvatarBlob.styles.js:49]: Did you modify something to get that? I am not seeing any issues when running it locally.

Comment (dsilva609) [reply templates/mobile/src/ui-kit/components/AvatarBlob/AvatarBlob.styles.js:49]: I just hardcoded it to see what it looks like if `leaf.r` returned 0

**Comment by jordanwade** [comment]
@calebpanza Looks like we might need a minimum size on the circles, the initials are breaking out of the circle at small sizes. I think something like that would address @dsilva609's feedback too.
image

**Comment by calebpanza** [comment]
> Looks like we might need a minimum size on the circles

I don't know about a minimum size, D3 is a physics based rendering and I'm not super confident about manipulating it beyond what it's already doing. Although, I think that we could do like "if the circle is less than `x`, don't show the initials` fairly easily.

> Should I see the little avatars in See what others are saying area?

Sorta! That part of the UI is gonna display other users that have _prayed_ for others. So if nobody has prayed for any prayers today, you wouldn't see any avatars.

**Comment by calebpanza** [comment]
@jordanwade I made some updates to both the rendering of the Initials and the animation duration. Check it out and lmk what you think

**Review by dsilva609 — APPROVED** [review]

**Pull request merged**